common/djangoapps/third_party_auth/tests/specs/test_testshib.py

```diff
@@ -109,6 +109,8 @@ def _configure_testshib_provider(self, **kwargs):
         kwargs.setdefault('icon_class', 'fa-university')
         kwargs.setdefault('attr_email', 'urn:oid:1.3.6.1.4.1.5923.1.1.1.6')  # eduPersonPrincipalName
         kwargs.setdefault('max_session_length', None)
+        kwargs.setdefault('send_to_registration_first', False)
```

Comment:
Do we have nay followup tasks here or are we all set, sorry I had lost track of this one?

Reply:
We should be all set. Default values being updated shouldn't* affect existing entries so ultimately this will just change how things look going forward.